honeybee_vtk/legend_parameter.py

```diff
@@ -571,6 +572,15 @@ def range(self) -> Tuple[float, float]:
                     f' value {self._min}.'
                 )
 
+    def _assign_colors(self, colors: Tuple[Color, Color]) -> None:
```

Comment:
From the code, it looks like you want a Tuple with variable-length but you are setting it here to a Tuple with two items.
https://docs.python.org/3/library/typing.html#typing.Tuple